rest-service/manager_rest/upload_manager.py

```diff
@@ -606,6 +606,56 @@ def _extract_application_file(cls,
         return application_file_name
 
 
+class UploadedBlueprintsValidator(UploadedBlueprintsManager):
+
+    @classmethod
+    def _prepare_and_submit_blueprint(cls,
+                                      file_server_root,
+                                      app_dir,
+                                      blueprint_id,
+                                      visibility):
+        args = get_args_and_verify_arguments([
+            Argument('application_file_name',
+                     default='')])
+
+        app_file_name = cls._extract_application_file(
+            file_server_root, app_dir, args.application_file_name)
+
+        # add to blueprints manager (will also dsl_parse it)
+        try:
+            get_resource_manager().validate_blueprint(
+                app_dir,
+                app_file_name,
+                file_server_root
+            )
+        except manager_exceptions.DslParseException, ex:
```

Comment:
this syntax is deprecated in 2.7 and removed in 3, please get used to writing this instead: `except manager_exceptions.DslParseException as ex:` (`as` instead of comma)

i realize there's other cases in this file of using this syntax, and there's no rush to change this now, because an upcoming py3 translation for cloudify-manager will handle this case automatically, just keep in mind that in the future this syntax won't work